### DISCUSSION TITLE
Is there any API Available to use the Custom Agents that I create in Lobe Chat? Or is there any plan for that ?

### DISCUSSION THREAD

**Discussion by ilovefreesw** [lobehub/lobehub#4746] [General]
Hi,
 
Just wanted to ask if there is a plan to use the custom agents over REST API or something like that?

Chatting on the web is not really usable for many use cases.......I want to use the agent I created in one of my web apps.

**Comment by arvinxx** [comment]
LobeChat designed for personal usage. So community edition is not aimed on service usage. But we plan to add restful API the next year in the paid edition. Stay tuned.

**Comment by ilovefreesw** [comment]
Oh ... then its useless for me either way.
